docs/sbpy/photometry.rst

```diff
@@ -96,37 +96,43 @@ model parameters:
       ---- ----
       3.31 0.12
 
-Note that model set is not supported.  Only one model can be initialized with
-the first set of valid parameters in the input `~sbpy.data.DataClass`.
+Note that in this case, model set is not supported.  Only one model can be
+initialized with the first set of valid parameters in the input
+`~sbpy.data.DataClass`.
 
 To fit a photometric model, one may follow the standard procedure defined in
 `astropy.modeling` submodule, first initializing a model, then using one of
 the fitter classes defined in `astropy.modeling.fitting`
-submodule, such as `~astropy.modeling.fitting.LevMarLSQFitter`.
+submodule, such as `~astropy.modeling.fitting.SLSQPLSQFitter`.  Note that
+the `~sbpy.photometry.HG1G2` model requires a constraint, 1 - G1 - G2 < 1,
+that is implemented in sbpy as an ineqcons-type constraint (see
```

Comment:
Could you write this more plainly, and move "ineqcons" to the remark?  e.g.:

"Note that  the `~sbpy.photometry.HG1G2` model requires an inequality constraint, 1 - G1 - G2 < 1, implemented in sbpy via the `ineqcons` parameter of `~astropy.modeling.Model`."

Reply:
updated.